build.go

```diff
@@ -233,202 +206,8 @@ func readVersionFromPackageJson() {
 	}
 }
 
-type linuxPackageOptions struct {
```

Comment:
@aknuds1 removed a bunch of stuff here that I think are not used anymore (especially the grunt package stuff as that was removed in PR a while back), and this linux package code is no longer used as far as I can tell (checked local Makefile)

Reply:
Thanks @torkelo :) It had to get cleaned up at some point.